projects/storefrontlib/src/cms-components/user/register/register.component.html

```diff
@@ -146,17 +146,18 @@
             </label>
           </div>
 
-          <div class="form-group">
+          <div class="form-group" *ngIf="anonymousConsent$ | async as consent">
             <div class="form-check">
               <label>
                 <input
                   type="checkbox"
                   name="newsletter"
                   class="form-check-input"
                   formControlName="newsletter"
+                  [checked]="isConsentGiven(consent.status)"
                 />
                 <span class="form-check-label">
-                  {{ 'register.emailMarketing' | cxTranslate }}
```

Comment:
I guess it's a breaking change to remove the `register.emailMarketing` translation.